content/docs/04.5-devx/05-app-profile/00-create-app-profile.md

```diff
@@ -14,118 +14,46 @@ import Tooltip from "shared/components/ui/Tooltip";
 
+5. Provide configuration information for the service you're adding.
 
+6. You can add more services to the App Profile if needed. To do this, click the **Actions** button next to the **Configure tier** pane. To rearrange layers, select a service and drag it up or down in the pane. Each service becomes a layer in the App Profile stack in the order shown in this pane.
 
-When you select a service, you will see the configuration options for that service on the righthand side of the screen. Once you have provided all the required information for the service you may click on the **Review** button to proceed. You may also add additional services or tiers, by clicking on the blue **Actions** button on the top lefthand of the screen. 
+7. When you've provided the required configuration information for services, click **Review**. 
 
-<br />
-
-
-5. **Review** the configuration and click on the **Finish Configuration** button.
-
-
-The Palette Dev Engine App Profile is now created and and can be used for Apps deployment. To edit or delete the App Profile, navigate to the **App Profile** page. Select the desired App Profile and click on the  **Settings** button at the top righthand side. 
-
-<br />
+Your App Profile is now created and can be used for Apps deployment. 
 
-## Validation
+# Validation
 
-You can validate the App Profile is available and ready for use by visiting the App Profiles page. In the App Profiles page, you will find your App Profile listed.
+To validate your App Profile is available and ready for use, navigate to the **App Profiles** page, where you'll find it listed.
 
-<br />
-
-## Security
-
-Palette encourages the developers to [securely distribute their credential using secrets](https://kubernetes.io/docs/tasks/inject-data-application/distribute-credentials-secure/). We can achieve security by constructing their manifests to consume the parameter by references. They can also add a manifest to create their Kubernetes secrets, using macros to inject the values. In the App tier, set the environment variables using a secret reference. 
-
-**Example Scenario**
-
-```
-apiVersion: v1
-kind: Secret
-metadata:
-  name: mysecret
-  namespace: {{.spectro.system.apptier.NAMESPACE}}
-data:
-  
-  namespace: "{{.spectro.system.apptier.NAMESPACE}}" # Resolves to a value of the variable defined in App Profile tier parameters.yaml file.
-
-  password: "{{.spectro.app.$appDepName-mongodb.PASSWORD}}" # To refer the tier output variables of the top tiers.
-	
-```
-```
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: myweb
-  namespace: mywebnamespace
-  ...
-spec:
-  template:
-  ...
-      containers:
-        - env:
-            - name: PASSWORD
-              valueFrom:
-                secretKeyRef:
-                  name: mysecret
-                  key: password
-```
-
-
-
-<br />
-<br />
\ No newline at end of file
+From this page, you can edit and delete App Profiles. Select an App Profile and click the **Settings** button. 
```

Comment:
I tried moving this up, but it feels right to me here because it follows the user workflow. It isn't until they navigate to the App Profiles page that they can view or modify Settings (edit/delete). I made it a single paragraph, though, and slightly reworded: "From this page, you can also edit and delete App Profiles in Settings."